project_template/static/src/js/modifiers.js

```diff
@@ -0,0 +1,69 @@
+/*
+    © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+odoo.define("project_template.hidden_fields", function(require) {
+"use strict";
+
+var Domain = require("web.Domain");
+
+function isNodeInvisibleOnTemplate(node){
+    var invisibleAttribute = node.attrs.invisible_on_template;
+    return invisibleAttribute && Boolean(eval(node.attrs.invisible_on_template));
+}
+
+function isProjectOrTask(state){
+    var model = state.model;
+    return model === "project.project" || model === "project.task";
+}
+
+require("web.FormRenderer").include({
+    /**
+     * Add a shortcut for the `project_template_invisible` modifier.
+     *
+     * The shortcut is to use the attribute `invisible_on_template`
+     * instead of attrs="{'project_template_invisible': [('is_template', '=', True)]}".
+     *
+     * <field name="my_field" invisible_on_template="1"/>
+     *
+     * This improves the readability and prevents having to rewrite the whole attrs
+     * for each node.
+     */
+    _renderFieldWidget(node, state) {
+        if(isProjectOrTask(state) && isNodeInvisibleOnTemplate(node)){
+            node.attrs.modifiers.project_template_invisible = "[('is_template', '=', True)]";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'project_template_invisible' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4397703)